Core/Object Arts/Dolphin/IDE/Base/ViewComposer.cls

```diff
@@ -78,15 +78,9 @@ adornmentRectanglesFor: aView
 	"Private - Answer a collection of rectangles that mark the edges of the secondary
```

Comment:
This method is not used and could just be deleted. No need to worry about that now though.